.github/workflows/screenshots.yml

```diff
@@ -0,0 +1,69 @@
+name: Visual Regression Tests
+
+on:
+  pull_request:
+    types: [labeled, synchronize, opened, reopened]
+    paths-ignore:
+      - 'docs/**'
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+  pull-requests: write
+
+jobs:
+  selenium-sqlite-screenshots:
+    if: contains(join(github.event.pull_request.labels.*.name, '|'), 'screenshots')
```

Comment:
It is because of the emoji (though I do love it) and I wasn't able to get it working by just putting the emoji into the string for some reason (I will commit this suggestion shortly to triple check)

Reply:
Might be worth checking how other projects that use emojis in labels handle this?

Reply:
I think there might be a hidden character at the end of the label (if you hover over the different labels with emojis I think there's something at the end of the screenshot label: https://github.com/django/django/labels). 
@felixxm can you check the label and maybe delete the extra character? 🙏

Reply:
> I think there might be a hidden character at the end of the label (if you hover over the different labels with emojis I think there's something at the end of the screenshot label: https://github.com/django/django/labels). @felixxm can you check the label and maybe delete the extra character? 🙏

There is nothing there (unfortunately).